ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14.yaml

```diff
@@ -74,6 +74,27 @@ tests:
     - chain: ipi-aws-pre
     - chain: workers-scale
     workflow: openshift-qe-node-density-heavy
+- as: ocp-qe-perfscale-ingress-perf-3nodes-periodic-aws
+  cron: 0 9 * * 3
+  steps:
+    cluster_profile: aws-perfscale-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "9"
```

Comment:
@krishvoor Can you try WORKER_REPLICA_COUNT instead of COMPUTE_NODE_REPLICAS? That should get past the worker count problem you were having.

Reply:
> @krishvoor Try WORKER_REPLICA_COUNT instead of COMPUTE_NODE_REPLICAS. That should get past the worker count problem you were having.

Sure, I am wondering if I have to pass COMPUTE_INSTANCE_TYPE, how can I control this behavior:
```
      COMPUTE_NODE_REPLICAS: "3"
      COMPUTE_NODE_TYPE: m5.2xlarge
      CONFIG: config/standard-classic-rosa.yml
      CONTROL_PLANE_INSTANCE_TYPE: r5.xlarge
      OPENSHIFT_INFRA_NODE_INSTANCE_TYPE: c5.4xlarge
      OPENSHIFT_INFRA_NODE_VOLUME_SIZE: "50"
      OPENSHIFT_INFRA_NODE_VOLUME_TYPE: gp3
      WORKER_REPLICA_COUNT: "9"
      LOKI_USE_SERVICEMONITOR	: "false"
      ZONES_COUNT: "3"

```

Reply:
I think I figured this one, closing this